### DISCUSSION TITLE
arm64 for macOS?

### DISCUSSION THREAD

**Discussion by eehlers** [MarkusJx/install-boost#33] [Q&A]
Does this installer support arm64 for MacOS?

In the documentation, under the `arch` parameter, it says "Only works on linux images" - so I am guessing not.

**Comment by MarkusJx** [comment]
This action only supports the operating systems and architectures of the github runners, and according to https://github.com/actions/runner-images/issues/2187, the github runners don't support arm. So no, it doesn't.

But I can add support for arm64 once the official github runners also support that

**Reply by eehlers** [reply]
Awesome, many thanks for getting back to me.

In case it's helpful, I managed to cross compile for arm using the runner for intel:

```
- name: Boost
  run: |
    curl -O -L https://boostorg.jfrog.io/artifactory/main/release/1.80.0/source/boost_1_80_0.tar.gz
    tar xfz boost_1_80_0.tar.gz
    cd boost_1_80_0
    ./bootstrap.sh
    ./b2 architecture=arm address-model=64 -s NO_LZMA=1 -s NO_ZSTD=1 abi=aapcs install
    lipo /usr/local/lib/libboost_*.dylib -info
```
